docs/tutorials/mobile/deploy-ios.md

```diff
@@ -110,13 +114,9 @@ This example is heavily based on [Google Tensorflow lite - Object Detection Exam
 
   It is written entirely in Swift and uses a bridgingheader file for the framework to be used in a Swift app.
 
-* The original TensorFlow tutorial and mode are [here])(https://www.tensorflow.org/lite/examples/object_detection/overview#get_started)
+* The original TensorFlow tutorial and model are [here])(https://www.tensorflow.org/lite/examples/object_detection/overview#get_started)
```

Comment:
link is not working here. it must be [here](url)

Reply:
Fixed